l10n_it_codici_carica/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,9 @@
+**Italiano**
+
+La tabella viene aggiunta nell'area *Fatturazione*, ed è raggiungibile nella 
+sezione configurazione. Occorre avere i privilegi di *Responsabile fatturazione*.
+
```

Comment:
Per uniformità con il resto meglio
```...nella sezione *Configurazione*.```

altrimenti
```....nella configurazione.```

Nelle versioni 11.0/12.0 è stata "nascosta" la parte di contabilità
In una installazione Odoo standard compaiono le etichette *Fatturazione*.
Dopo aver installato i moduli contabili OCA diventa *Contabilità*.

Dato che teoricamente questo modulo potrebbe essere usato in entrambe le situazioni (solo fatturazione o con moduli di contabilità) forse sarebbe meglio usare
```*Fatturazione/Contabilità*```

@eLBati @tafaRU cosa ne pensate?

Reply:
> forse sarebbe meglio usare
`*Fatturazione/Contabilità*`

:+1:

Reply:
Ok @primes2h.
@mymage riesci ad aggiornare la PR?

Reply:
All done

Reply:
Thanks @mymage !
@primes2h ti chiederei un'ultima review prima di mergiare. Grazie.